Formula/ktmpl.rb

```diff
@@ -0,0 +1,19 @@
+class Ktmpl < Formula
+  desc "Parameterized templates for Kubernetes manifests."
+  homepage "https://github.com/InQuicker/ktmpl"
+  url "https://github.com/InQuicker/ktmpl/archive/0.5.0.tar.gz"
+  sha256 "36924798cb88ddc98f43fea33aff894297cf4068a00157e08bf53373a79bb12c"
+  head "https://github.com/InQuicker/ktmpl.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+
+    bin.install "target/release/ktmpl"
+  end
+
+  test do
+    assert_match "ktmpl #{version}", shell_output("#{bin}/ktmpl --version")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Consider generating a template. Thanks!